python/dataclean.py

```diff
@@ -90,6 +90,35 @@ def addSeason(cursor, seasonsData, qualiResultsData, qualiChanges, year):
     seasonsData[year] = s
     return no_mistakes
 
+def addRaceSeasonData(cursor, seasonsData, raceResultsData, year):
+    """Adds a Season Data object to the given map of seasons"""
+    s = Season()
+
+    sql = "SELECT `raceId`, `round`, `circuitId`, `name` FROM `races` WHERE `year`=%s"
+    cursor.execute(sql, year)
+    result = cursor.fetchall()
+    
+    for x in result:
+        circuitId = x.get('circuitId')
+        roundNo = x.get('round')
+
+        raceData = RaceData(circuitId, roundNo)
+        s.addRace(x.get('raceId'), raceData)
+        res = addRaceResults(cursor, raceResultsData, x.get('raceId'))
```

Comment:
`res` here is unnecessary as `addRaceResults()` doesn't return anything (in the quali model it was used to detect whether this race had results available or not).